### PR TITLE
[modtools] Better UI for initial linking

### PR DESCRIPTION
**Pull request by phil-lopreiato**
Don't just 403 `/mod` if there are no keys. Clean up useless docs page and add some more details to the redeem copy instead

### CODE REVIEW

File: templates_jinja2/team_admin_redeem.html [hunk 1]
```diff
@@ -35,7 +35,9 @@ <h1>Redeem Team Admin Code</h1>
   </ul>
   <a class="btn btn-primary" href="/mod"><span class="glyphicon glyphicon-eye-open"></span> View Mod Dashboard</a>
 {% else %}
-  <p>Redeem an access code to moderate a team's media. More help <a href="/mod/help">can be found here</a>.</p>
+  <p>Redeem an access code to moderate a team's media. Having moderator access allows you to manage media linked to the team's TBA profile as well as accept/reject suggestions for additional media.</p>
```
Comment (fangeugene) [inline templates_jinja2/team_admin_redeem.html]: What about "...moderate your team's media" instead of "a"?

Comment (phil-lopreiato) [reply templates_jinja2/team_admin_redeem.html]: :+1:

File: templates_jinja2/team_admin_redeem.html [hunk 2]
```diff
@@ -35,7 +35,7 @@ <h1>Redeem Team Admin Code</h1>
   </ul>
   <a class="btn btn-primary" href="/mod"><span class="glyphicon glyphicon-eye-open"></span> View Mod Dashboard</a>
 {% else %}
-  <p>Redeem an access code to moderate a team's media. Having moderator access allows you to manage media linked to the team's TBA profile as well as accept/reject suggestions for additional media.</p>
+  <p>Redeem an access code to moderate you team's media on TBA. Having moderator access allows you to manage media linked to your team's TBA profile as well as accept/reject suggestions for additional media links.</p>
```
Comment (fangeugene) [inline templates_jinja2/team_admin_redeem.html]: YOUUU

Comment (phil-lopreiato) [reply templates_jinja2/team_admin_redeem.html]: 😠

**Comment by phil-lopreiato** [comment]
omg, github rendered 😠 correctly as a commit message :heart:

**Pull request merged**